### PR TITLE
Accept single string as command in the okteto manifest

### PR DESCRIPTION
**Pull request by pchico83**
Single strings, when possible, improve readability and usability of okteto manifests

### CODE REVIEW

File: pkg/model/serializer.go [hunk 1]
```diff
@@ -47,6 +47,38 @@ func (e EnvVar) MarshalYAML() (interface{}, error) {
...
+func (c *Command) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var multi []string
+	err := unmarshal(&multi)
+	if err != nil {
+		var single string
+		err := unmarshal(&single)
+		if err != nil {
+			return err
+		}
+		if strings.Contains(single, " ") {
```
Comment (rberrelleza) [inline pkg/model/serializer.go:60]: What is this check for? I think doing `strings.Split` would work better here.

File: pkg/model/serializer.go [hunk 2]
```diff
@@ -47,6 +47,38 @@ func (e EnvVar) MarshalYAML() (interface{}, error) {
...
+	}
+	return nil
+}
+
+// MarshalYAML Implements the marshaler interface of the yaml pkg.
+func (c Command) MarshalYAML() (interface{}, error) {
+	if len(c.Values) == 1 && !strings.Contains(c.Values[0], " ") {
+		return c.Values[0], nil
+	}
+	if len(c.Values) == 3 && c.Values[0] == "sh" && c.Values[1] == "-c" && !strings.Contains(c.Values[2], " ") {
```
Comment (rberrelleza) [inline pkg/model/serializer.go:76]: Having the `sh -c` on the command is odd. I think that's something we should add during the execution, but not the manifest.

**Comment by rberrelleza** [comment]
> Single strings, when possible, improve readability and usability of okteto manifests

Don't this break with how docker and kubernetes takes commands though?

File: pkg/model/serializer.go (hunk 1 above)
Comment (pchico83) [reply pkg/model/serializer.go:60]: I don't want to split, it is just a checking to know if it is a single command or not

File: pkg/model/serializer.go (hunk 2 above)
Comment (pchico83) [reply pkg/model/serializer.go:76]: It is the other way around. It is to remove them when marshaling. Although thinking twice, I don't need this even for the `init` command

**Pull request merged**